VideoCalls/AuthenticationViewController.m

```diff
@@ -100,9 +100,9 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
         for (NSString *component in components)
         {
             if ([component hasPrefix:userPrefix])
-                user = [component substringFromIndex:[userPrefix length]];
+                user = [[component substringFromIndex:[userPrefix length]] stringByRemovingPercentEncoding];
             if ([component hasPrefix:passPrefix])
-                token = [component substringFromIndex:[passPrefix length]];
+                token = [[component substringFromIndex:[passPrefix length]] stringByRemovingPercentEncoding];
         }
         
         NSLog(@"SERVER:%@ USER:%@ TOKEN:%@", _serverUrl, user, token);
```

Comment:
Please remove logging :)

Reply:
Yeah, I think it's time to remove that log 😄